hedera-node/src/main/java/com/hedera/services/grpc/marshalling/AliasResolver.java

```diff
@@ -225,24 +223,70 @@ private Result resolveInternalFungible(
             }
             final var resolution = aliasManager.lookupIdBy(alias);
             if (resolution == MISSING_NUM) {
-                result = netOf(isEvmAddress, alias);
+                if (isForToken) {
```

Comment:
I guess since this is already unreadable, we can just do,
```
result = netOf(isEvmAddress, alias, !isForToken);
```
🤷

Reply:
Fixed !